lib/viewing/RenderableView.js

```diff
@@ -58,10 +54,38 @@ var RenderableView = _.extend({},
             }.bind(this));
         },
 
+        generateViewPlaceholders: function() {
+            var unplacedCount = this.childViewCount() - this.unrenderedChildViewCount();
+
+            // [wawjr3d] Do this before return so data.views is always an object
+            this.viewPlaceholders = new Array(unplacedCount);
+
+            if (!unplacedCount) {
```

Comment:
maybe this line can go up to line 59 and this.viewPlaceholders can be set to an empty array by default?